doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -3171,6 +3173,8 @@ Syntax:
 Return the expression value after casting to the specified
 :ref:`data type <sql_column_def_data_type>`.
 
+Warning: CAST to/from UUID may change some components to/from little-endian.
```

Comment:
Though VARBINARY representation of UUID may be not exactly as we expect, it doesn't mean that it is wrong. Besides, `CAST(CAST(<varbinary> AS UUID) AS VARBINARY)` works as expected:
```
tarantool> box.execute([[select hex(cast(cast(x'12345678123412340fffffffffffffff' as UUID) as VARBINARY));]])
---
- metadata:
  - name: COLUMN_1
    type: string
  rows:
  - ['12345678123412340FFFFFFFFFFFFFFF']
...
```

Reply:
Right, I should not have used a word that made it seem that something was wrong. I believe I have resolved this by removing the word "Warning:".